src/field.rs

```diff
@@ -232,7 +226,8 @@ mod tests {
                 -1.488542499264307,
                 0.19211668257573278,
                 2.3427520079106143
-            ])
+            ]),
+            max_ulps = 7
```

Comment:
Not sure if accuracy really decreased. My guess is that `Zip::fold` will just sum things with slightly different associations due to avoiding in-place accumulation, i.e. `sum + ...` instead of `sum += ...`.